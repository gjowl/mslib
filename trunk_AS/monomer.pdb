ATOM      1  N   ALA A  23      -4.567  -4.302 -17.899  1.00  0.00           N
ATOM      2  CA  ALA A  23      -3.581  -4.652 -16.888  1.00  0.00           C
ATOM      3  HA  ALA A  23      -3.989  -5.372 -16.188  1.00  0.00           H
ATOM      4  HT1 ALA A  23      -4.821  -5.150 -18.444  1.00  0.00           H
ATOM      5  HT2 ALA A  23      -4.169  -3.585 -18.539  1.00  0.00           H
ATOM      6  HT3 ALA A  23      -5.417  -3.920 -17.437  1.00  0.00           H
ATOM      7  CB  ALA A  23      -2.344  -5.306 -17.547  1.00  0.00           C
ATOM      8  HB1 ALA A  23      -2.655  -6.219 -18.098  1.00  0.00           H
ATOM      9  HB2 ALA A  23      -1.867  -4.614 -18.275  1.00  0.00           H
ATOM     10  HB3 ALA A  23      -1.586  -5.603 -16.791  1.00  0.00           H
ATOM     11  C   ALA A  23      -3.190  -3.453 -16.081  1.00  0.00           C
ATOM     12  O   ALA A  23      -3.051  -3.532 -14.862  1.00  0.00           O
ATOM     13  N   ALA A  24      -3.000  -2.302 -16.755  1.00  0.00           N
ATOM     14  HN  ALA A  24      -3.090  -2.217 -17.747  1.00  0.00           H
ATOM     15  CA  ALA A  24      -2.640  -1.077 -16.054  1.00  0.00           C
ATOM     16  HA  ALA A  24      -1.740  -1.224 -15.467  1.00  0.00           H
ATOM     17  CB  ALA A  24      -2.346   0.055 -17.065  1.00  0.00           C
ATOM     18  HB1 ALA A  24      -1.515  -0.250 -17.735  1.00  0.00           H
ATOM     19  HB2 ALA A  24      -3.236   0.266 -17.698  1.00  0.00           H
ATOM     20  HB3 ALA A  24      -2.047   0.995 -16.552  1.00  0.00           H
ATOM     21  C   ALA A  24      -3.712  -0.675 -15.091  1.00  0.00           C
ATOM     22  O   ALA A  24      -3.430  -0.262 -13.968  1.00  0.00           O
ATOM     23  N   ALA A  25      -4.987  -0.790 -15.516  1.00  0.00           N
ATOM     24  HN  ALA A  25      -5.242  -1.110 -16.428  1.00  0.00           H
ATOM     25  CA  ALA A  25      -6.098  -0.443 -14.643  1.00  0.00           C
ATOM     26  HA  ALA A  25      -5.999   0.575 -14.284  1.00  0.00           H
ATOM     27  CB  ALA A  25      -7.436  -0.528 -15.413  1.00  0.00           C
ATOM     28  HB1 ALA A  25      -7.412   0.167 -16.280  1.00  0.00           H
ATOM     29  HB2 ALA A  25      -7.608  -1.554 -15.803  1.00  0.00           H
ATOM     30  HB3 ALA A  25      -8.297  -0.245 -14.770  1.00  0.00           H
ATOM     31  C   ALA A  25      -6.119  -1.318 -13.429  1.00  0.00           C
ATOM     32  O   ALA A  25      -6.354  -0.849 -12.317  1.00  0.00           O
ATOM     33  N   ALA A  26      -5.872  -2.629 -13.620  1.00  0.00           N
ATOM     34  HN  ALA A  26      -5.692  -3.054 -14.505  1.00  0.00           H
ATOM     35  CA  ALA A  26      -5.849  -3.555 -12.496  1.00  0.00           C
ATOM     36  HA  ALA A  26      -6.782  -3.511 -11.946  1.00  0.00           H
ATOM     37  CB  ALA A  26      -5.683  -5.007 -12.999  1.00  0.00           C
ATOM     38  HB1 ALA A  26      -6.523  -5.265 -13.679  1.00  0.00           H
ATOM     39  HB2 ALA A  26      -4.735  -5.127 -13.567  1.00  0.00           H
ATOM     40  HB3 ALA A  26      -5.686  -5.735 -12.159  1.00  0.00           H
ATOM     41  C   ALA A  26      -4.769  -3.195 -11.524  1.00  0.00           C
ATOM     42  O   ALA A  26      -4.971  -3.236 -10.313  1.00  0.00           O
ATOM     43  N   TYR A  27      -3.581  -2.829 -12.045  1.00  0.00           N
ATOM     44  HN  TYR A  27      -3.373  -2.796 -13.021  1.00  0.00           H
ATOM     45  CA  TYR A  27      -2.477  -2.440 -11.179  1.00  0.00           C
ATOM     46  HA  TYR A  27      -2.307  -3.250 -10.481  1.00  0.00           H
ATOM     47  CB  TYR A  27      -1.191  -2.102 -11.975  1.00  0.00           C
ATOM     48  HB1 TYR A  27      -1.470  -1.557 -12.902  1.00  0.00           H
ATOM     49  HB2 TYR A  27      -0.539  -1.433 -11.372  1.00  0.00           H
ATOM     50  CG  TYR A  27      -0.343  -3.294 -12.337  1.00  0.00           C
ATOM     51  CD1 TYR A  27       0.185  -4.150 -11.360  1.00  0.00           C
ATOM     52  HD1 TYR A  27      -0.041  -3.980 -10.317  1.00  0.00           H
ATOM     53  CD2 TYR A  27      -0.005  -3.522 -13.675  1.00  0.00           C
ATOM     54  HD2 TYR A  27      -0.386  -2.861 -14.441  1.00  0.00           H
ATOM     55  CE1 TYR A  27       1.010  -5.227 -11.724  1.00  0.00           C
ATOM     56  HE1 TYR A  27       1.407  -5.893 -10.971  1.00  0.00           H
ATOM     57  CE2 TYR A  27       0.820  -4.587 -14.044  1.00  0.00           C
ATOM     58  HE2 TYR A  27       1.054  -4.747 -15.087  1.00  0.00           H
ATOM     59  CZ  TYR A  27       1.322  -5.434 -13.071  1.00  0.00           C
ATOM     60  OH  TYR A  27       2.130  -6.496 -13.489  1.00  0.00           O
ATOM     61  HH  TYR A  27       2.470  -6.283 -14.362  1.00  0.00           H
ATOM     62  C   TYR A  27      -2.826  -1.226 -10.375  1.00  0.00           C
ATOM     63  O   TYR A  27      -2.522  -1.149  -9.188  1.00  0.00           O
ATOM     64  N   LEU A  28      -3.483  -0.240 -11.017  1.00  0.00           N
ATOM     65  HN  LEU A  28      -3.736  -0.256 -11.982  1.00  0.00           H
ATOM     66  CA  LEU A  28      -3.885   0.970 -10.313  1.00  0.00           C
ATOM     67  HA  LEU A  28      -2.989   1.358  -9.847  1.00  0.00           H
ATOM     68  CB  LEU A  28      -4.475   2.089 -11.222  1.00  0.00           C
ATOM     69  HB1 LEU A  28      -5.472   1.774 -11.593  1.00  0.00           H
ATOM     70  HB2 LEU A  28      -4.625   2.999 -10.598  1.00  0.00           H
ATOM     71  CG  LEU A  28      -3.609   2.479 -12.434  1.00  0.00           C
ATOM     72  HG  LEU A  28      -3.581   1.609 -13.130  1.00  0.00           H
ATOM     73  CD1 LEU A  28      -4.226   3.676 -13.190  1.00  0.00           C
ATOM     74 HD11 LEU A  28      -3.600   3.939 -14.068  1.00  0.00           H
ATOM     75 HD12 LEU A  28      -5.248   3.425 -13.543  1.00  0.00           H
ATOM     76 HD13 LEU A  28      -4.288   4.559 -12.521  1.00  0.00           H
ATOM     77  CD2 LEU A  28      -2.167   2.800 -12.022  1.00  0.00           C
ATOM     78 HD21 LEU A  28      -1.568   3.076 -12.916  1.00  0.00           H
ATOM     79 HD22 LEU A  28      -2.155   3.652 -11.311  1.00  0.00           H
ATOM     80 HD23 LEU A  28      -1.683   1.925 -11.541  1.00  0.00           H
ATOM     81  C   LEU A  28      -4.845   0.654  -9.209  1.00  0.00           C
ATOM     82  O   LEU A  28      -4.747   1.201  -8.113  1.00  0.00           O
ATOM     83  N   LEU A  29      -5.807  -0.250  -9.479  1.00  0.00           N
ATOM     84  HN  LEU A  29      -5.931  -0.708 -10.358  1.00  0.00           H
ATOM     85  CA  LEU A  29      -6.775  -0.637  -8.462  1.00  0.00           C
ATOM     86  HA  LEU A  29      -7.215   0.282  -8.099  1.00  0.00           H
ATOM     87  CB  LEU A  29      -7.928  -1.552  -8.972  1.00  0.00           C
ATOM     88  HB1 LEU A  29      -7.524  -2.558  -9.207  1.00  0.00           H
ATOM     89  HB2 LEU A  29      -8.664  -1.675  -8.144  1.00  0.00           H
ATOM     90  CG  LEU A  29      -8.684  -1.033 -10.208  1.00  0.00           C
ATOM     91  HG  LEU A  29      -7.980  -1.049 -11.072  1.00  0.00           H
ATOM     92  CD1 LEU A  29      -9.879  -1.949 -10.548  1.00  0.00           C
ATOM     93 HD11 LEU A  29     -10.408  -1.572 -11.448  1.00  0.00           H
ATOM     94 HD12 LEU A  29      -9.530  -2.983 -10.748  1.00  0.00           H
ATOM     95 HD13 LEU A  29     -10.596  -1.974  -9.701  1.00  0.00           H
ATOM     96  CD2 LEU A  29      -9.163   0.412 -10.016  1.00  0.00           C
ATOM     97 HD21 LEU A  29      -9.701   0.757 -10.924  1.00  0.00           H
ATOM     98 HD22 LEU A  29      -9.853   0.473  -9.150  1.00  0.00           H
ATOM     99 HD23 LEU A  29      -8.308   1.097  -9.843  1.00  0.00           H
ATOM    100  C   LEU A  29      -6.095  -1.281  -7.294  1.00  0.00           C
ATOM    101  O   LEU A  29      -6.432  -1.014  -6.143  1.00  0.00           O
ATOM    102  N   LEU A  30      -5.107  -2.155  -7.570  1.00  0.00           N
ATOM    103  HN  LEU A  30      -4.811  -2.411  -8.488  1.00  0.00           H
ATOM    104  CA  LEU A  30      -4.373  -2.815  -6.499  1.00  0.00           C
ATOM    105  HA  LEU A  30      -5.117  -3.278  -5.865  1.00  0.00           H
ATOM    106  CB  LEU A  30      -3.388  -3.925  -6.970  1.00  0.00           C
ATOM    107  HB1 LEU A  30      -2.529  -3.457  -7.495  1.00  0.00           H
ATOM    108  HB2 LEU A  30      -2.985  -4.437  -6.066  1.00  0.00           H
ATOM    109  CG  LEU A  30      -3.994  -4.997  -7.894  1.00  0.00           C
ATOM    110  HG  LEU A  30      -4.264  -4.506  -8.858  1.00  0.00           H
ATOM    111  CD1 LEU A  30      -2.967  -6.109  -8.196  1.00  0.00           C
ATOM    112 HD11 LEU A  30      -3.412  -6.868  -8.873  1.00  0.00           H
ATOM    113 HD12 LEU A  30      -2.066  -5.683  -8.683  1.00  0.00           H
ATOM    114 HD13 LEU A  30      -2.659  -6.611  -7.255  1.00  0.00           H
ATOM    115  CD2 LEU A  30      -5.271  -5.602  -7.300  1.00  0.00           C
ATOM    116 HD21 LEU A  30      -5.685  -6.369  -7.989  1.00  0.00           H
ATOM    117 HD22 LEU A  30      -5.047  -6.086  -6.326  1.00  0.00           H
ATOM    118 HD23 LEU A  30      -6.048  -4.826  -7.144  1.00  0.00           H
ATOM    119  C   LEU A  30      -3.646  -1.816  -5.653  1.00  0.00           C
ATOM    120  O   LEU A  30      -3.627  -1.921  -4.429  1.00  0.00           O
ATOM    121  N   ALA A  31      -3.022  -0.810  -6.299  1.00  0.00           N
ATOM    122  HN  ALA A  31      -3.010  -0.711  -7.294  1.00  0.00           H
ATOM    123  CA  ALA A  31      -2.309   0.221  -5.560  1.00  0.00           C
ATOM    124  HA  ALA A  31      -1.558  -0.221  -4.915  1.00  0.00           H
ATOM    125  CB  ALA A  31      -1.572   1.169  -6.534  1.00  0.00           C
ATOM    126  HB1 ALA A  31      -0.854   0.586  -7.150  1.00  0.00           H
ATOM    127  HB2 ALA A  31      -2.287   1.668  -7.224  1.00  0.00           H
ATOM    128  HB3 ALA A  31      -1.000   1.952  -5.990  1.00  0.00           H
ATOM    129  C   ALA A  31      -3.237   0.983  -4.667  1.00  0.00           C
ATOM    130  O   ALA A  31      -2.907   1.285  -3.522  1.00  0.00           O
ATOM    131  N   VAL A  32      -4.440   1.313  -5.180  1.00  0.00           N
ATOM    132  HN  VAL A  32      -4.742   1.099  -6.105  1.00  0.00           H
ATOM    133  CA  VAL A  32      -5.419   2.035  -4.380  1.00  0.00           C
ATOM    134  HA  VAL A  32      -4.923   2.928  -4.019  1.00  0.00           H
ATOM    135  CB  VAL A  32      -6.632   2.493  -5.223  1.00  0.00           C
ATOM    136  HB  VAL A  32      -6.247   3.119  -6.062  1.00  0.00           H
ATOM    137  CG1 VAL A  32      -7.358   1.287  -5.847  1.00  0.00           C
ATOM    138 HG11 VAL A  32      -8.223   1.636  -6.451  1.00  0.00           H
ATOM    139 HG12 VAL A  32      -6.679   0.716  -6.511  1.00  0.00           H
ATOM    140 HG13 VAL A  32      -7.740   0.609  -5.055  1.00  0.00           H
ATOM    141  CG2 VAL A  32      -7.594   3.387  -4.398  1.00  0.00           C
ATOM    142 HG21 VAL A  32      -8.452   3.710  -5.026  1.00  0.00           H
ATOM    143 HG22 VAL A  32      -7.996   2.834  -3.525  1.00  0.00           H
ATOM    144 HG23 VAL A  32      -7.065   4.291  -4.033  1.00  0.00           H
ATOM    145  C   VAL A  32      -5.826   1.236  -3.181  1.00  0.00           C
ATOM    146  O   VAL A  32      -5.959   1.771  -2.082  1.00  0.00           O
ATOM    147  N   LEU A  33      -6.036  -0.082  -3.370  1.00  0.00           N
ATOM    148  HN  LEU A  33      -5.954  -0.554  -4.246  1.00  0.00           H
ATOM    149  CA  LEU A  33      -6.414  -0.943  -2.260  1.00  0.00           C
ATOM    150  HA  LEU A  33      -7.284  -0.485  -1.807  1.00  0.00           H
ATOM    151  CB  LEU A  33      -6.796  -2.398  -2.663  1.00  0.00           C
ATOM    152  HB1 LEU A  33      -5.884  -2.941  -2.990  1.00  0.00           H
ATOM    153  HB2 LEU A  33      -7.184  -2.918  -1.758  1.00  0.00           H
ATOM    154  CG  LEU A  33      -7.855  -2.519  -3.773  1.00  0.00           C
ATOM    155  HG  LEU A  33      -7.409  -2.129  -4.718  1.00  0.00           H
ATOM    156  CD1 LEU A  33      -8.246  -3.995  -4.004  1.00  0.00           C
ATOM    157 HD11 LEU A  33      -9.000  -4.070  -4.815  1.00  0.00           H
ATOM    158 HD12 LEU A  33      -7.355  -4.592  -4.290  1.00  0.00           H
ATOM    159 HD13 LEU A  33      -8.679  -4.426  -3.078  1.00  0.00           H
ATOM    160  CD2 LEU A  33      -9.103  -1.684  -3.462  1.00  0.00           C
ATOM    161 HD21 LEU A  33      -9.846  -1.790  -4.280  1.00  0.00           H
ATOM    162 HD22 LEU A  33      -9.566  -2.031  -2.515  1.00  0.00           H
ATOM    163 HD23 LEU A  33      -8.850  -0.608  -3.365  1.00  0.00           H
ATOM    164  C   LEU A  33      -5.344  -0.970  -1.212  1.00  0.00           C
ATOM    165  O   LEU A  33      -5.631  -0.922  -0.018  1.00  0.00           O
ATOM    166  N   PHE A  34      -4.071  -1.046  -1.645  1.00  0.00           N
ATOM    167  HN  PHE A  34      -3.799  -1.100  -2.605  1.00  0.00           H
ATOM    168  CA  PHE A  34      -2.961  -1.054  -0.702  1.00  0.00           C
ATOM    169  HA  PHE A  34      -3.132  -1.876  -0.019  1.00  0.00           H
ATOM    170  CB  PHE A  34      -1.623  -1.233  -1.457  1.00  0.00           C
ATOM    171  HB1 PHE A  34      -1.711  -2.078  -2.170  1.00  0.00           H
ATOM    172  HB2 PHE A  34      -1.398  -0.314  -2.040  1.00  0.00           H
ATOM    173  CG  PHE A  34      -0.457  -1.537  -0.553  1.00  0.00           C
ATOM    174  CD1 PHE A  34      -0.501  -2.618   0.340  1.00  0.00           C
ATOM    175  HD1 PHE A  34      -1.389  -3.229   0.407  1.00  0.00           H
ATOM    176  CD2 PHE A  34       0.714  -0.768  -0.628  1.00  0.00           C
ATOM    177  HD2 PHE A  34       0.763   0.067  -1.313  1.00  0.00           H
ATOM    178  CE1 PHE A  34       0.604  -2.926   1.143  1.00  0.00           C
ATOM    179  HE1 PHE A  34       0.558  -3.762   1.826  1.00  0.00           H
ATOM    180  CE2 PHE A  34       1.822  -1.073   0.170  1.00  0.00           C
ATOM    181  HE2 PHE A  34       2.719  -0.473   0.103  1.00  0.00           H
ATOM    182  CZ  PHE A  34       1.767  -2.155   1.056  1.00  0.00           C
ATOM    183  HZ  PHE A  34       2.622  -2.393   1.674  1.00  0.00           H
ATOM    184  C   PHE A  34      -2.922   0.216   0.091  1.00  0.00           C
ATOM    185  O   PHE A  34      -2.691   0.198   1.298  1.00  0.00           O
ATOM    186  N   THR A  35      -3.150   1.362  -0.582  1.00  0.00           N
ATOM    187  HN  THR A  35      -3.326   1.424  -1.564  1.00  0.00           H
ATOM    188  CA  THR A  35      -3.154   2.643   0.108  1.00  0.00           C
ATOM    189  HA  THR A  35      -2.195   2.733   0.605  1.00  0.00           H
ATOM    190  CB  THR A  35      -3.328   3.816  -0.868  1.00  0.00           C
ATOM    191  HB  THR A  35      -4.363   3.802  -1.280  1.00  0.00           H
ATOM    192  OG1 THR A  35      -2.396   3.706  -1.945  1.00  0.00           O
ATOM    193  HG1 THR A  35      -2.891   3.736  -2.769  1.00  0.00           H
ATOM    194  CG2 THR A  35      -3.082   5.156  -0.155  1.00  0.00           C
ATOM    195 HG21 THR A  35      -3.203   5.997  -0.868  1.00  0.00           H
ATOM    196 HG22 THR A  35      -3.801   5.302   0.677  1.00  0.00           H
ATOM    197 HG23 THR A  35      -2.051   5.186   0.255  1.00  0.00           H
ATOM    198  C   THR A  35      -4.237   2.693   1.141  1.00  0.00           C
ATOM    199  O   THR A  35      -4.030   3.187   2.246  1.00  0.00           O
ATOM    200  N   LEU A  36      -5.432   2.178   0.793  1.00  0.00           N
ATOM    201  HN  LEU A  36      -5.647   1.783  -0.097  1.00  0.00           H
ATOM    202  CA  LEU A  36      -6.540   2.162   1.738  1.00  0.00           C
ATOM    203  HA  LEU A  36      -6.658   3.182   2.081  1.00  0.00           H
ATOM    204  CB  LEU A  36      -7.901   1.699   1.139  1.00  0.00           C
ATOM    205  HB1 LEU A  36      -7.856   0.611   0.919  1.00  0.00           H
ATOM    206  HB2 LEU A  36      -8.689   1.848   1.913  1.00  0.00           H
ATOM    207  CG  LEU A  36      -8.344   2.433  -0.139  1.00  0.00           C
ATOM    208  HG  LEU A  36      -7.632   2.163  -0.954  1.00  0.00           H
ATOM    209  CD1 LEU A  36      -9.756   1.983  -0.572  1.00  0.00           C
ATOM    210 HD11 LEU A  36     -10.059   2.509  -1.501  1.00  0.00           H
ATOM    211 HD12 LEU A  36      -9.774   0.890  -0.760  1.00  0.00           H
ATOM    212 HD13 LEU A  36     -10.493   2.218   0.224  1.00  0.00           H
ATOM    213  CD2 LEU A  36      -8.304   3.955   0.037  1.00  0.00           C
ATOM    214 HD21 LEU A  36      -8.626   4.456  -0.901  1.00  0.00           H
ATOM    215 HD22 LEU A  36      -8.988   4.262   0.856  1.00  0.00           H
ATOM    216 HD23 LEU A  36      -7.279   4.304   0.277  1.00  0.00           H
ATOM    217  C   LEU A  36      -6.207   1.337   2.941  1.00  0.00           C
ATOM    218  O   LEU A  36      -6.509   1.719   4.070  1.00  0.00           O
ATOM    219  N   LEU A  37      -5.568   0.172   2.723  1.00  0.00           N
ATOM    220  HN  LEU A  37      -5.318  -0.182   1.824  1.00  0.00           H
ATOM    221  CA  LEU A  37      -5.184  -0.689   3.833  1.00  0.00           C
ATOM    222  HA  LEU A  37      -6.084  -0.858   4.409  1.00  0.00           H
ATOM    223  CB  LEU A  37      -4.616  -2.078   3.416  1.00  0.00           C
ATOM    224  HB1 LEU A  37      -3.614  -1.944   2.957  1.00  0.00           H
ATOM    225  HB2 LEU A  37      -4.478  -2.687   4.339  1.00  0.00           H
ATOM    226  CG  LEU A  37      -5.492  -2.883   2.440  1.00  0.00           C
ATOM    227  HG  LEU A  37      -5.508  -2.341   1.466  1.00  0.00           H
ATOM    228  CD1 LEU A  37      -4.897  -4.287   2.194  1.00  0.00           C
ATOM    229 HD11 LEU A  37      -5.530  -4.852   1.479  1.00  0.00           H
ATOM    230 HD12 LEU A  37      -3.873  -4.206   1.776  1.00  0.00           H
ATOM    231 HD13 LEU A  37      -4.848  -4.853   3.147  1.00  0.00           H
ATOM    232  CD2 LEU A  37      -6.937  -3.001   2.938  1.00  0.00           C
ATOM    233 HD21 LEU A  37      -7.544  -3.584   2.213  1.00  0.00           H
ATOM    234 HD22 LEU A  37      -6.962  -3.520   3.918  1.00  0.00           H
ATOM    235 HD23 LEU A  37      -7.405  -2.001   3.050  1.00  0.00           H
ATOM    236  C   LEU A  37      -4.215   0.005   4.739  1.00  0.00           C
ATOM    237  O   LEU A  37      -4.318  -0.085   5.960  1.00  0.00           O
ATOM    238  N   LEU A  38      -3.239   0.724   4.150  1.00  0.00           N
ATOM    239  HN  LEU A  38      -3.111   0.812   3.165  1.00  0.00           H
ATOM    240  CA  LEU A  38      -2.263   1.453   4.949  1.00  0.00           C
ATOM    241  HA  LEU A  38      -1.836   0.732   5.633  1.00  0.00           H
ATOM    242  CB  LEU A  38      -1.097   2.088   4.134  1.00  0.00           C
ATOM    243  HB1 LEU A  38      -1.487   2.937   3.534  1.00  0.00           H
ATOM    244  HB2 LEU A  38      -0.357   2.504   4.855  1.00  0.00           H
ATOM    245  CG  LEU A  38      -0.353   1.128   3.188  1.00  0.00           C
ATOM    246  HG  LEU A  38      -1.060   0.827   2.380  1.00  0.00           H
ATOM    247  CD1 LEU A  38       0.854   1.830   2.529  1.00  0.00           C
ATOM    248 HD11 LEU A  38       1.374   1.132   1.840  1.00  0.00           H
ATOM    249 HD12 LEU A  38       0.520   2.716   1.952  1.00  0.00           H
ATOM    250 HD13 LEU A  38       1.574   2.162   3.306  1.00  0.00           H
ATOM    251  CD2 LEU A  38       0.104  -0.143   3.913  1.00  0.00           C
ATOM    252 HD21 LEU A  38       0.634  -0.815   3.205  1.00  0.00           H
ATOM    253 HD22 LEU A  38       0.796   0.118   4.740  1.00  0.00           H
ATOM    254 HD23 LEU A  38      -0.761  -0.699   4.331  1.00  0.00           H
ATOM    255  C   LEU A  38      -2.929   2.500   5.785  1.00  0.00           C
ATOM    256  O   LEU A  38      -2.593   2.684   6.952  1.00  0.00           O
ATOM    257  N   SER A  39      -3.904   3.220   5.195  1.00  0.00           N
ATOM    258  HN  SER A  39      -4.201   3.115   4.246  1.00  0.00           H
ATOM    259  CA  SER A  39      -4.623   4.248   5.936  1.00  0.00           C
ATOM    260  HA  SER A  39      -3.890   4.945   6.322  1.00  0.00           H
ATOM    261  CB  SER A  39      -5.635   4.988   5.041  1.00  0.00           C
ATOM    262  HB1 SER A  39      -6.530   4.349   4.870  1.00  0.00           H
ATOM    263  HB2 SER A  39      -5.972   5.927   5.532  1.00  0.00           H
ATOM    264  OG  SER A  39      -5.058   5.276   3.774  1.00  0.00           O
ATOM    265  HG1 SER A  39      -4.412   5.982   3.898  1.00  0.00           H
ATOM    266  C   SER A  39      -5.363   3.655   7.094  1.00  0.00           C
ATOM    267  O   SER A  39      -5.377   4.215   8.187  1.00  0.00           O
ATOM    268  N   LEU A  40      -6.005   2.490   6.874  1.00  0.00           N
ATOM    269  HN  LEU A  40      -6.032   2.009   6.000  1.00  0.00           H
ATOM    270  CA  LEU A  40      -6.733   1.825   7.946  1.00  0.00           C
ATOM    271  HA  LEU A  40      -7.420   2.560   8.344  1.00  0.00           H
ATOM    272  CB  LEU A  40      -7.564   0.585   7.500  1.00  0.00           C
ATOM    273  HB1 LEU A  40      -6.875  -0.242   7.230  1.00  0.00           H
ATOM    274  HB2 LEU A  40      -8.169   0.242   8.370  1.00  0.00           H
ATOM    275  CG  LEU A  40      -8.521   0.824   6.318  1.00  0.00           C
ATOM    276  HG  LEU A  40      -7.904   1.024   5.411  1.00  0.00           H
ATOM    277  CD1 LEU A  40      -9.380  -0.430   6.043  1.00  0.00           C
ATOM    278 HD11 LEU A  40     -10.056  -0.250   5.181  1.00  0.00           H
ATOM    279 HD12 LEU A  40      -8.732  -1.300   5.811  1.00  0.00           H
ATOM    280 HD13 LEU A  40      -9.998  -0.673   6.932  1.00  0.00           H
ATOM    281  CD2 LEU A  40      -9.422   2.042   6.553  1.00  0.00           C
ATOM    282 HD21 LEU A  40     -10.098   2.189   5.684  1.00  0.00           H
ATOM    283 HD22 LEU A  40     -10.041   1.888   7.462  1.00  0.00           H
ATOM    284 HD23 LEU A  40      -8.822   2.966   6.681  1.00  0.00           H
ATOM    285  C   LEU A  40      -5.812   1.443   9.064  1.00  0.00           C
ATOM    286  O   LEU A  40      -6.145   1.601  10.236  1.00  0.00           O
ATOM    287  N   ALA A  41      -4.618   0.925   8.717  1.00  0.00           N
ATOM    288  HN  ALA A  41      -4.330   0.773   7.772  1.00  0.00           H
ATOM    289  CA  ALA A  41      -3.648   0.543   9.734  1.00  0.00           C
ATOM    290  HA  ALA A  41      -4.083  -0.164  10.431  1.00  0.00           H
ATOM    291  CB  ALA A  41      -2.429  -0.150   9.082  1.00  0.00           C
ATOM    292  HB1 ALA A  41      -2.765  -1.053   8.530  1.00  0.00           H
ATOM    293  HB2 ALA A  41      -1.925   0.526   8.357  1.00  0.00           H
ATOM    294  HB3 ALA A  41      -1.685  -0.470   9.843  1.00  0.00           H
ATOM    295  C   ALA A  41      -3.224   1.730  10.543  1.00  0.00           C
ATOM    296  O   ALA A  41      -3.096   1.646  11.763  1.00  0.00           O
ATOM    297  N   ALA A  42      -2.993   2.874   9.870  1.00  0.00           N
ATOM    298  HN  ALA A  42      -3.075   2.961   8.878  1.00  0.00           H
ATOM    299  CA  ALA A  42      -2.599   4.086  10.574  1.00  0.00           C
ATOM    300  HA  ALA A  42      -1.708   3.911  11.167  1.00  0.00           H
ATOM    301  CB  ALA A  42      -2.262   5.208   9.565  1.00  0.00           C
ATOM    302  HB1 ALA A  42      -1.435   4.876   8.901  1.00  0.00           H
ATOM    303  HB2 ALA A  42      -3.139   5.447   8.926  1.00  0.00           H
ATOM    304  HB3 ALA A  42      -1.936   6.137  10.081  1.00  0.00           H
ATOM    305  C   ALA A  42      -3.665   4.523  11.531  1.00  0.00           C
ATOM    306  O   ALA A  42      -3.376   4.926  12.655  1.00  0.00           O
ATOM    307  N   ALA A  43      -4.939   4.449  11.097  1.00  0.00           N
ATOM    308  HN  ALA A  43      -5.197   4.137  10.183  1.00  0.00           H
ATOM    309  CA  ALA A  43      -6.045   4.831  11.963  1.00  0.00           C
ATOM    310  HA  ALA A  43      -5.916   5.846  12.323  1.00  0.00           H
ATOM    311  CB  ALA A  43      -7.379   4.789  11.184  1.00  0.00           C
ATOM    312  HB1 ALA A  43      -7.328   5.483  10.318  1.00  0.00           H
ATOM    313  HB2 ALA A  43      -7.581   3.768  10.793  1.00  0.00           H
ATOM    314  HB3 ALA A  43      -8.235   5.099  11.822  1.00  0.00           H
ATOM    315  C   ALA A  43      -6.102   3.957  13.178  1.00  0.00           C
ATOM    316  OT1 ALA A  43      -5.235   3.050  13.295  1.00  0.00           O
ATOM    317  OT2 ALA A  43      -7.014   4.174  14.020  1.00  0.00           O
TER     318      ALA A  43                                                    
END
