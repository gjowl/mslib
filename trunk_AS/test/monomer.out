Monomer - VDW weight: 1 HB weight: 1 IMM1REF weight: 1 IMM1 weight: 1
